DeviceTests/Caboodle.DeviceTests.Android/Caboodle.DeviceTests.Android.csproj

```diff
@@ -90,6 +90,9 @@
       <SubType>Designer</SubType>
     </AndroidResource>
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
Where did this come from?

Reply:
I think VS adds it if you open the test explorer. It somehow feels that it needs to, i just remove it before committing.